modules/picard_gc_bias_metrics.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description": "GC bias statistics gather by the Picard CollectGcBiasMetrics tool",
   "type": "object",
-  "minProperties": 10,
+  "minProperties": 11,
```

Comment:
no please not set this :(. keeping it on 10 keeps is possible to still use the old version